eng/pipelines/runtime.yml

```diff
@@ -964,6 +964,47 @@ extends:
                     eq(variables['monoContainsChange'], true),
                     eq(variables['isRollingBuild'], true))
 
+      #
+      # Android devices
```

Comment:
Devices or emulators?

Reply:
This should be x64 emulator and arm64 device.